app/views/shipments/show.jbuilder

```diff
@@ -0,0 +1,20 @@
+json.shipment do 
+  json.company_id           @shipment.company.id
+  json.destination_country  @shipment.destination_country
+  json.origin_country       @shipment.origin_country
+  json.tracking_number      @shipment.tracking_number
+  json.slug                 @shipment.slug
+  json.created_at           @shipment.created_at.strftime('%A, %d %B %Y at%l:%M %p')
+  json.items                @shipment.group_items
+
+  if @tracker.present?
+    json.tracker do
+      json.status                   @tracker.status
+      json.current_location         @tracker.current_location
+      json.last_checkpoint_message  @tracker.last_checkpoint_message
+      json.last_checkpoint_time     @tracker.last_checkpoint_time
```

Comment:
This is the structure we would like for our response but this structure does not match Aftership's response body. You would have to parse Aftership's response (see example response body below) and extract the relevant information:

![image](https://user-images.githubusercontent.com/5158383/102802603-ce554f80-43ae-11eb-9fe0-a29c9334d9bd.png)

For example, for `status`, you would need to use `tracker['data']['tag']`.

Reply:
Yes I have actually fixed my initial issue I read through the API website again and realised the names of Aftership's response body were different to what I had so I adjusted my show.jbuilder file. It now returns all the data as null, I believe this is due to supplying the api with a fake tracking number so it won't return any information and instead will return null.